src/gamedata/decallib.h

```diff
@@ -84,6 +84,8 @@ class FDecalTemplate : public FDecalBase
 	const FDecalBase *LowerDecal;
 
 	enum { DECAL_RandomFlipX = 0x100, DECAL_RandomFlipY = 0x200 };
+
+	bool opaqueBlood;
```

Comment:
Please put new member after `RenderFlags`, so the size of objects remains unchanged.
Alternatively, I can create a new decal-specific render flag for this purpose, but it’s not as trivial as moving a member.

Reply:
@alexey-lysiuk did as you asked... is this what you meant?

Reply:
Yes, that's better.